digiwf-libs/digiwf-spring-security/digiwf-spring-security-core/src/main/java/de/muenchen/oss/digiwf/spring/security/client/OAuth2AccessTokenSupplier.java

```diff
@@ -16,32 +19,37 @@
  * Supplier for service account access token using given registration.
  */
 @Component
+@ConditionalOnProperty(value = "digiwf.security.service-account", matchIfMissing = true)
```

Comment:
do wee need this @ConditionalOnProperty? Service-Account-Auth is always configured in the application.yml and the OAuth2AccessTokenSupplier bean has to be actively used.

Reply:
This is true for the engine and task service only. If we integrate it to another service, the service account is potentially not required.